### ISSUE TITLE
3 featrue requests

### ISSUE THREAD

**Issue by cser** [Dadido3/batch-rename#1]
Thank you for this utility! If you want to improve it, there exists only 3 problems:
- the most important - lack of the file filter, for example: `batch-rename *.odt;*.txt;*.docx;`
- the --hide-numbers option is missing - numbers prevent you from editing numbers in file names by search-replace;
- it would be nice to rename folders too.

**Comment by Dadido3** [comment]
Thanks for the ideas.

Most of these shouldn't be too hard to implement, so i may take a try in the next few days.

I'm just not sure about renaming/moving folders, as this can have side effects for any following entries that are inside the moved folder. But i could add an option that will automatically create and remove (empty) directories as needed.

**Comment by cser** [comment]
> But i could add an option that will automatically create and remove (empty) directories as needed.

Yes, I went through my ideas on the logic of renaming for 20 minutes and came to the same thing O_o.
